api/cluster/templater.go

```diff
@@ -296,7 +300,7 @@ func (t *KFServingResourceTemplater) createTransformerSpec(modelService *models.
 			{Name: transformerpkg.JaegerSamplerType, Value: jaegerCfg.SamplerType},
 			{Name: transformerpkg.JaegerDisabled, Value: jaegerCfg.Disabled},
 		}
-		envVars = append(jaegerEnvVars, envVars...)
+		envVars = append(envVars, jaegerEnvVars...)
```

Comment:
why you change the ordering? initially the purpose of put `jaegerEnvVars` first to let user overwrite the value of jaeger envs

Reply:
No particular reason. At first, I got confused why my test failed, turns out the envVars order is different with the order of the code written. I also forgot about these ordering but I think moving forward we should use filtering for those allowed-to-be-modified env vars.

Reply:
what kind of filtering? with this change user can't enable the jaeger tracing

Reply:
Filtering on UI: https://github.com/gojek/merlin/blob/HEAD/ui/src/components/EnvVarsConfigTable.js#L41

Why can user enable the jaeger tracing ya if the order changed?

Reply:
currently the default env variable JAEGER_DISABLED is set to true, and user wants to set JAEGER_DISABLED to false, user will add JAEGER_DISABLED value in env variables to false when about to deploy. So  initial `envVars` has `JAEGER_DISABLED` false because it is set by user. And later this `jaegerEnvVars` value is appended to `envVars`. Let say `jaegerEnvVars` only has [`JAEGER_DISABLED`: true] after appended the `envVars` will have value [`JAEGER_DISABLED`: false,  `JAEGER_DISABLED`: true] . The value that win will be the last one right, which is the default value?

Reply:
Understood. Thanks for detailed explanation. I've update my PR to revert the order so Jaeger env vars is on top of the slice.

On another note, I think we should patch the `JAEGER_DISABLED` with user-provided value (if exist) instead of having two env var with the same name. It's out of scope of this PR and we should refactor it in the future.